routing/index_router.cpp

```diff
@@ -1016,4 +1028,24 @@ RouterResultCode IndexRouter::ConvertTransitResult(set<NumMwmId> const & mwmIds,
 
   return RouterResultCode::TransitRouteNotFoundTooLongPedestrian;
 }
+
+void IndexRouter::FillSpeedCamProhibitedMwms(vector<Segment> const & segments,
+                                             vector<platform::CountryFile> & speedCamProhibitedMwms) const
+{
+  CHECK(m_numMwmIds, ());
+
+  set<NumMwmId> mwmIds;
+  for (auto const & s : segments)
+    mwmIds.insert(s.GetMwmId());
```

Comment:
nit: I'd rather use emplace wherever it's possible

Reply:
For `uint16_t`? :) Done.